bin/test

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def main
+  begin
+    command = create_command
+  rescue ArgumentError => e
+    puts e.message
+    exit
+  end
+  puts "Running #{command}"
+  system(command)
+end
+
+def create_command
+  case ARGV.length
+  when 0
+    "bundle exec rake test"
+  when 1
+    filename = ARGV[0]
+    "bundle exec rake test TEST=#{filename}"
+  when 2
+    filename = ARGV[0]
+    test_name = ARGV[1]
+    test_name_with_underscores = test_name.tr(" ", "_")
+    test_name_pattern = "/#{Regexp.escape(test_name_with_underscores)}/"
+    test_name_pattern_escaped_single_quotes = test_name_pattern.gsub("'", %q(\\\'))
+    "bundle exec rake test TEST=#{filename} TESTOPTS=\"--name=#{test_name_pattern_escaped_single_quotes} -v\""
```

Comment:
Would we have to worry about this escaping if we constructed the command as an array to pass to `system`?